rocAL/rocAL_pybind/amd/rocAL/fn.py

```diff
@@ -128,7 +128,7 @@ def gamma_correction(*inputs, gamma=0.5, device=None):
     gamma_correction_image = b.GammaCorrection(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
     return (gamma_correction_image)
 
-def hue(*inputs, bytes_per_sample_hint=0,  hue=0.0, image_type=0,
+def hue(*inputs, bytes_per_sample_hint=0,  hue=None, image_type=0,
```

Comment:
Why the hue=None here?

Reply:
That is a default value Rajy
Later a FloatParam is created if a float value is passed by the user
else, a None would passed to the C API call indicating the default value will be picked
hue = b.CreateFloatParameter(hue) if isinstance(hue, float) else hue